EdFi.Buzz.Installer/eng/README.md

```diff
@@ -0,0 +1,105 @@
+# Project Buzz Installer
+
+These scripts were made possible thanks to the Ed-Fi Alliance and the Michael and Susan Dell Foundation.
+
+## Description
+
+The instalation script downloads all the assets needed for the Ed-Fi Buzz. Artifacts are placed in the dist/ directory at the root of this project 
```

Comment:
Just see a typo - instalation > installation. I would also note that the UI project uses build/ as its root.